src/arrayMethodSort.js

```diff
@@ -4,9 +4,24 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultCompare) {
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = 0; j < this.length - i - 1; j++) {
```

Comment:
Good job, but we can reduce the complexity of the algorithm. You don't need to do the next iteration in the outer loop if there are no changes in the previous one. Consider using do while loop with a counter variable for detecting changes. Feel free to write in the chat, if you have questions